src/components/TimeGutter.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component, PropTypes } from 'react'
+import dates from '../utils/dates';
+import TimeSliceGroup from './TimeSliceGroup.jsx'
+import TimeSlice from './TimeSlice.jsx'
+import SelectableTimeSlice from '../containers/SelectableTimeSlice.jsx'
+
+export default class TimeGutter extends Component {
+  static propTypes = {
+    step: PropTypes.number.isRequired,
+    slices: PropTypes.number.isRequired,
+    now: PropTypes.instanceOf(Date).isRequired,
+    min: PropTypes.instanceOf(Date).isRequired,
+    max: PropTypes.instanceOf(Date).isRequired,
+    hidelabels: PropTypes.bool,
+    timesliceClassnames: PropTypes.string,
+    timeGutterFormat: PropTypes.string
+  }
+  static defaultProps = {
+    step: 10,
+    slices: 2,
+    selectable: false,
+    timesliceClassnames: '',
```

Comment:
we don't expose classnames this way in other places do we? It feels a bit out of place to allow that here. If anything it should be a getter function to allow styling as well